src/content/events/2024/bytesize_metatdenovo.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Bytesize: nf-core/metatdenovo'
+subtitle: Danilo Di Leo, Linné­ universitetet
```

Comment:
```suggestion
subtitle: Danilo Di Leo, Linné­universitetet
```
Sorry, my bad, had to double check and it's actually one word 🤯

Reply:
:angry: